<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QWidget" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>471</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(86, 118, 162);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>143</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Ubuntu Mono</family>
             <pointsize>24</pointsize>
            </font>
           </property>
           <property name="text">
            <string>欢迎使用智能大棚系统</string>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QPushButton" name="passwd_verify">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Ubuntu Mono</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
    background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(149, 187, 209, 255), stop:1 rgba(218, 238, 249, 255));
    border-style: outset;
    border-radius: 8px;
	color:rgb(86, 118, 162);
    }
QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(17, 100, 162, 255), stop:1 rgba(61, 140, 201, 255));
    border-style: inset;
    border-radius: 8px;
	color:white;
    }</string>
             </property>
             <property name="text">
              <string>密码验证</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="face_verify">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Ubuntu Mono</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
    background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(149, 187, 209, 255), stop:1 rgba(218, 238, 249, 255));
    border-style: outset;
    border-radius: 8px;
	color:rgb(86, 118, 162);
    }
QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(17, 100, 162, 255), stop:1 rgba(61, 140, 201, 255));
    border-style: inset;
    border-radius: 8px;
	color:white;
    }</string>
             </property>
             <property name="text">
              <string>人脸验证</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="turn_off">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Ubuntu Mono</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
    background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(149, 187, 209, 255), stop:1 rgba(218, 238, 249, 255));
    border-style: outset;
    border-radius: 8px;
	color:rgb(86, 118, 162);
    }
QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(17, 100, 162, 255), stop:1 rgba(61, 140, 201, 255));
    border-style: inset;
    border-radius: 8px;
	color:white;
    }</string>
             </property>
             <property name="text">
              <string>关门</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="init_verify">
             <property name="minimumSize">
              <size>
               <width>200</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>200</width>
               <height>50</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Ubuntu Mono</family>
               <pointsize>15</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
    background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(149, 187, 209, 255), stop:1 rgba(218, 238, 249, 255));
    border-style: outset;
    border-radius: 8px;
	color:rgb(86, 118, 162);
    }
QPushButton:pressed {
    background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(17, 100, 162, 255), stop:1 rgba(61, 140, 201, 255));
    border-style: inset;
    border-radius: 8px;
	color:white;
    }</string>
             </property>
             <property name="text">
              <string>系统初始化</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>143</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
